che-code-sshd.yaml

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
I assume this file should actually be from this PR and proposed into https://github.com/eclipse-che/che-operator/tree/main/editors-definitions ?

Reply:
I'll go ahead and do this. I've addressed the other issues (and fixed the sshd.pid file, to be written correctly).

Reply:
Ok, removed it from this PR and re-located to https://github.com/eclipse-che/che-operator/pull/2039 .